l10n_it_fatturapa_in/tests/data/IT05979361218_001.xml

```diff
@@ -67,6 +67,10 @@
                 </DatiCassaPrevidenziale>
                 <Causale>Rif ordine MAPA: --- Nr. Identificativo Ordine 1234567</Causale>
             </DatiGeneraliDocumento>
+            <DatiTrasporto>
+                <PesoLordo>0.00</PesoLordo>
+                <PesoNetto>0.00</PesoNetto>
+            </DatiTrasporto>
```

Comment:
Riassumento: ```PesoLordo``` può essere presente senza ```DatiAnagraficiVettore```. Quindi in questo caso bisogna cmq scrivere il campo ```gross_weight``` della fattura.
Stesso discorso per ```PesoNetto```